File: swagger/models/get_sites_request.py
from .base_model import BaseModel


class GetSitesRequest(BaseModel):
	"""
	no description available

	Attributes:

		site_ids
		List of the requested site IDs. When omitted, returns all sites that
		the source has access to.

		limit
		Number of results to include, defaults to 100

		offset
		Page offset, defaults to 0.
	"""

	input_map = {
		'SiteIds': 'site_ids',
		'Limit': 'limit',
		'Offset': 'offset',
		}

	output_map = {
		'site_ids': 'SiteIds',
		'limit': 'Limit',
		'offset': 'Offset',
		}

File: swagger/Models/GetSitesRequest.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetSitesRequest extends BaseModel
{
	/**
	 * List of the requested site IDs. When omitted, returns all sites that
	 * the source has access to.
	 * 
	 * @var integer[] 
	 */
	public $siteIds;
	/**
	 * Number of results to include, defaults to 100
	 * 
	 * @var integer format:int32
	 */
	public $limit;
	/**
	 * Page offset, defaults to 0.
	 * 
	 * @var integer format:int32
	 */
	public $offset;

	protected function getInputMap()
	{
		return [
			'SiteIds' => ['siteIds', 'array', null],
			'Limit' => ['limit', 'integer', 'int32'],
			'Offset' => ['offset', 'integer', 'int32'],
			];
	}

	protected function getOutputMap()
	{
		return [
			'siteIds' => ['SiteIds', 'array', null],
			'limit' => ['Limit', 'integer', 'int32'],
			'offset' => ['Offset', 'integer', 'int32'],
			];
	}
}
